.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,11 +19,13 @@
 
 ## Checklist
 
-- [ ] :ok_hand: design updates are Garden Designer approved (add the
-      designer as a reviewer)
+<!-- check the items below that will be completed prior to merge.
+     strikethrough any item text that does not apply to this PR. -->
+
+- [ ] :ok_hand: design updates will be Garden Designer approved (add the designer as a reviewer)
```

Comment:
Why the switch to future tense?

Reply:
We want the PR author to check these items when the PR is submitted for review (aka, "I've thought of the items needed to open a PR"). We've seen folks confused about holding off on checkmarks because they haven't secured design review.